spec/classes/init_spec.rb

```diff
@@ -2375,6 +2375,12 @@
         it { should compile.with_all_deps }
 
         it { should contain_class('ssh') }
```

Comment:
wrapped this in an exported_resource context so the test does not fail as it seems that the sshkey resource is always present just not exported if we set the `ssh_key_export` to false.